### PR TITLE
deps: updates to golang 1.19.3

### PR DESCRIPTION
**Pull request by LanceEa**
## Description

Updates to Golang 1.19.3 to make sure we stay up on the latest z patches. 

## Related Issues

N/A - normal dependency updates.

## Testing

CI

## Checklist

- [x] **Does my change need to be backported to a previous release?**
  _Yes, will want to backport to release/v2.5 branch to keep it updated as well_
- [x] **I made sure to update `CHANGELOG.md`.**
- [x] **This is unlikely to impact how Ambassador performs at scale.**
- [x] **My change is adequately tested.**
- [ ] **I updated `DEVELOPING.md` with any any special dev tricks I had to use to work on this code efficiently.**
- [x] **The changes in this PR have been reviewed for security concerns and adherence to security best practices.**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -155,3 +155,11 @@ python-dev-setup:
...
 # activate venv
 	@echo "run 'source ./venv/bin/activate' to activate venv in local shell"
+
+# re-generate docs
+.PHONY: clean-changelog
+clean-changelog:
+	rm CHANGELOG.md
+
+.PHONY: generate-changelog
+generate-changelog: clean-changelog $(PWD)/CHANGELOG.md
```
Comment (haq204) [inline Makefile:165]: is the recipe to be filled in later?

Comment (LanceEa) [reply Makefile:165]: No, left empty on purpose and just leveraging dependent targets to force the behavior.

 We have the target `$(PWD)/CHANGELOG.md` but the only way to run it today is part of the larger `generate` or `generate-fast`. I wanted a way to quickly make changes. For example, when needing to fix a merge conflict. In this scenario, I know that all I need to do is trigger this single step to run. 

So, `clean-changelog` is a new target that just removes the file. Not sure if there is a better way to do that. Then added `$(PWD)/CHANGELOG.md` target to re-create it.

**Review by haq204 — COMMENTED** [review]
looks good. just need to rebase once https://github.com/emissary-ingress/emissary/pull/4666 lands

**Comment by LanceEa** [comment]
Will re-run `make generate` looks like the deps file didn't update so will get that addressed.

**Comment by LanceEa** [comment]
Alright, should be good to review and ready to go once green.

**Review by haq204 — APPROVED** [review]

**Pull request merged**